dist/pinny.js

```diff
@@ -443,6 +445,19 @@
             return $(document.activeElement);
         },
 
+        /*
+        * In iOS, there is an issue when soft keyboard triggers, position fixed
+        * elements on screen don't get updated, distorting the UI.
+        * Fix: blur keyboard on orientationchange
+        */
+        _blurOnOrientationChange: function($pinny) {
+            if ($.os.ios) {
+                $(window).on('orientationchange', function(e) {
+                    $pinny.find(':focus').blur();
```

Comment:
@ericmuyser document.activeElement is not always an input and _hideSpacer is only hiding the spacer when activeElement is not an input. This way, we will not have a 'jump' when navigating from 1 input to another